creating_packages/toolchains.rst

```diff
@@ -10,49 +10,62 @@ Toolchains
 
     Please try it and provide feedback at: https://github.com/conan-io/conan/issues
 
+.. warning:
+
+    Starting in Conan 1.32 ``write_toolchain_files()`` method and ``toolchain`` attribute have been
```

Comment:
```suggestion
    Starting in Conan 1.32 ``toolchain()`` method and ``toolchain`` attribute have been
```

Reply:
But, we are deprecating also the write_toolchain_files, shouldn't we mention that?

Reply:
Sure, but probably it is more related to the classes, when we are talking about the class CMakeToolchain, MakeToolchain,... this section is more focused on the ConanFile methods/attribs, right?